### PR TITLE
Swift 5 support

### PR DESCRIPTION
**Pull request by fassko**
Adding RxSwift 5.0.x support

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Current master
 
 - Nothing yet!
 
+4.5.0
```
Comment (freak4pc) [inline Changelog.md]: This needs to be 5.0.0

**Review by freak4pc — CHANGES_REQUESTED** [review]

File: Changelog.md (hunk 1 above)
Comment (ashfurrow) [inline Changelog.md]:
```suggestion
5.0.0
```

Comment (fassko) [reply Changelog.md]: Thanks, I changed in podspec as well.

**Review by freak4pc — COMMENTED** [review]
LGTM!

**Comment by freak4pc** [comment]
@fassko Thanks for you work here! I just realized this is one of the only projects where we didn't update to Circle v2 from v1 - so CI just doesn't run anymore. Is there any chance you could find the time to do that? It should be mostly straightforward

**Comment by fassko** [comment]
@freak4pc Sure I will create an issue and do it. I have done it for some of my projects.

**Comment by fassko** [comment]
I just discovered that there is an issue created by me :D

https://github.com/RxSwiftCommunity/NSObject-Rx/issues/62

**Comment by freak4pc** [comment]
_self-five to past @fassko_

**Pull request merged**